lint/lint.go

```diff
@@ -17,6 +18,11 @@ type LintEngine interface {
 	IgnoreRule(rule string)
 }
 
+// export the function NewEngine to be used in other packages
+func NewEngine(rootDir string, source []byte) (*internal.Engine, error) {
+	return internal.NewEngine(rootDir, source)
+}
```

Comment:
Since this `NewEngine` only serves the role of running the linter, it might be better to change its name to `New`, if you don't have any plan to add another initializers.

And also, please modify the `main.go` file where `internal.NewEngine` is used.

https://github.com/gnolang/tlin/blob/49c75b0bee93dfd5c05b59565bc8ae013bf66f92/cmd/tlin/main.go#L53